custom/aaa/tests/test_child_beneficiary_view.py

```diff
@@ -17,7 +17,15 @@ def setUpClass(cls):
             domain=cls.domain,
             person_case_id='person_case_id',
             child_health_case_id='child_health_case_id',
-            opened_on='2019-01-01'
+            opened_on='2019-01-01',
+            breastfed_within_first='yes',
```

Comment:
is there a reason these aren't booleans?

Reply:
So for the CAS dashboard it was often (still is) very frustrating to have to go through multiple layers of data transformation to come back with a query that is consistent with the app level definitions. Because of this any new work I do waits until the very last moment to make any changes to the data. This generally means:

1. If there is a change in how the app saves questions, we can more quickly respond to it in reports if we save the raw value. For example sometimes they care about whether the question was yes/no, sometimes yes/no/skipped. Making as few assumptions as possible reduces mental load for me.
2. UCR data sources only use property_path or property_name
3. UCR data sources keep the same names as the form question or case property
4. Django tables should keep the same column name unless the meaning of it changes. For example, when we aggregate to a location table (i.e. one row per AWC) an regisitered_eligible_couple is an idea that doesn't exist in the app, so it has a name that isn't directly coorelated with the app.

In this case `breastfed_within_first` is saved from a single choice with values 'yes'/no. It could also be null if the form wasn't filled out and the property doesn't exist.